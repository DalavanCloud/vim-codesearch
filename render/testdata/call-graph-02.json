{
    "call_graph_response": [
        {
            "node": {
                "call_scope_range": {
                    "start_line": 0, 
                    "end_line": 0, 
                    "start_column": 0, 
                    "end_column": 0
                }, 
                "call_site_range": {
                    "start_line": 0, 
                    "end_line": 0, 
                    "start_column": 0, 
                    "end_column": 0
                }, 
                "snippet": {
                    "text": {}, 
                    "first_line_number": 0, 
                    "match_reason": {
                        "blame": false, 
                        "content": false, 
                        "filename_lineno": false, 
                        "scoped_symbol": false, 
                        "filename": false
                    }
                }, 
                "node_kind": "FUNCTION", 
                "signature": "kythe://chromium?lang=c%2B%2B?path=src/net/http/http_auth_handler_negotiate_unittest.cc#K4ikCIS9mEQ2mHfCRGMW5LQewkaTRwkdAaRECIIEiX8%3D", 
                "override": false
            }, 
            "is_from_kythe": true, 
            "return_code": 1, 
            "results_offset": 0, 
            "estimated_total_number_results": 0, 
            "is_call_graph": true
        }
    ]
}
