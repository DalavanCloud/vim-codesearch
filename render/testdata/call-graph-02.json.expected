This file contains the rendered output per line and its associated metadata.

Lines that begin with a line number contains (after the '|') the contents that
will be inserted into the vim buffer at that line. The immediately following
line contains an object representing associated metadata.
-------------------------------------------------------------------------------

001|^N{[+] 
   |{'s': 'kythe://chromium?lang=c%2B%2B?path=src/net/http/http_auth_handler_negotiate_unittest.cc#K4ikCIS9mEQ2mHfCRGMW5LQewkaTRwkdAaRECIIEiX8%3D'}
002|
   |{}
003|}N_
   |{}
